File: 프로그래머스/0/181836. 그림 확대/그림 확대.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> picture, int k) {
    vector<string> answer;
    for(int i = 0; i < k * picture.size(); i++){
        string str = "";
        for(int j = 0; j < k * picture[0].size(); j++){
            str += picture[i / k][j / k];
        }
        answer.push_back(str);
    }
    return answer;
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.js
function solution(picture, k) {
    let answer = [];

    for (let i = 0; i < k * picture.length; i++) {
        let str = "";
        for (let j = 0; j < k * picture[0].length; j++) {
            str += picture[Math.floor(i / k)][Math.floor(j / k)];
        }
        answer.push(str);
    }

    return answer;
}
